### PR TITLE
feat: modules create at start

### PR DESCRIPTION
**Pull request by ghasemdev**

### CODE REVIEW

**Comment by ghasemdev** [comment]
@arnaudgiuliani

**Comment by arnaudgiuliani** [comment]
Super interesting thanks 👍 Let me check your PR

**Comment by ghasemdev** [comment]
> Super interesting thanks 👍 Let me check your PR

thanks

File: compiler/koin-ksp-compiler/src/jvmMain/kotlin/org/koin/compiler/verify/KoinConfigVerification.kt [hunk 1]
```diff
@@ -24,7 +24,12 @@ import org.koin.compiler.generator.getFile
 import org.koin.compiler.metadata.KoinMetaData
 import java.io.OutputStream
 
-private val ignored = listOf("kotlin.Any",)
+private val ignored = listOf(
+    "kotlin.Any",
+    "androidx.lifecycle.SavedStateHandle",
+    "android.content.Context",
+    "android.app.Application"
+)
```
Comment (arnaudgiuliani) [inline compiler/koin-ksp-compiler/src/jvmMain/kotlin/org/koin/compiler/verify/KoinConfigVerification.kt:32]: same comment here :) need to remove this, as it's already part of another PR

**Review by arnaudgiuliani — CHANGES_REQUESTED** [review]
Just remove the ignored part

**Comment by arnaudgiuliani** [comment]
Need to prepare branch for 1.4.0, and I'll merge it

**Review by arnaudgiuliani — APPROVED** [review]

**Pull request merged**